ST_mortality.c

```diff
@@ -1378,6 +1382,47 @@ void killMaxage(void) {
     }
 }
 
+/**
+ * \brief Conduct quantile mapping to convert a three-year annual or
+ * perennial biomass average from satellite to STEPWAT2 biomass
```

Comment:
@N1ckP3rsl3y @MartinHoldrege suggest we expand this explanation a bit to add a few more details as provided in biomass_qm.in.

Also, can we upload full documentation to Github that contains the workflow as an Github issue? Didn't see one. Thanks